backend/settings.py

```diff
@@ -85,8 +85,17 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'NAME': 'user_data',
+        'ENGINE': 'django.db.backends.mysql',
+        'HOST': '127.0.0.1',
+        'PORT': 3306,
+        'USER': 'root',
+        'PASSWORD': 'lmaozedong',  # change this to env variable maybe
+        'OPTIONS': {
+          'autocommit': True,
+          'use_oure': True,
+          'init_command': "SET foo='bar';"
```

Comment:
Is there a follow-up task which captures setting a real `init_command`? I think this should also have a `TODO`.